### DISCUSSION TITLE
Mass Importing TGs

### DISCUSSION THREAD

**Discussion by tadscottsmith** [chuot/rdio-scanner#112] [Q&A]
Is there a new way to import TGs (similar to the load-tr command) in the new version? I have close to 5000 and doing anything in the admin interface is painful with that number.

**Comment by chuot** [comment]
In short, no

The idea/concept behind Rdio Scanner was never intended to handle so many talkgroups.

The admin interface since v6.1 warns you if you try to import more than 800 talkgroups into a system.

Not only is the admin interface will be slow, but the main scanner interface will queue new calls too rapidely, forcing you to avoid many talkgroups, scrolling through the select tg panel will be a pain, and so on...

These super nets with lots of talkgroups are usually spread over many sites where not every site has them all.

A simple solution would be to take advantage of the autopopulate option for this system. Configure your trunk-recorder with a talkgroupsfile containing the 5000+ talkgroups. Only talkgroups with activities will be uploaded to Rdio Scanner and the talkgroup labels will be populated according to what's in the talkgroupsfile on the trunk-recorder side. In the end, I'm sure you won't get more than 200, but I could be wrong.

**Comment by tadscottsmith** [comment]
Thanks - I just wanted to make sure nothing existed before I wrote something. I understand it's probably not a real common use case. While listening live to all of them is impossible, it's great to be able to playback specific TGs after an incident occurs. Thanks for your work!

**Reply by chuot** [reply]
Again, you could let the autopopulate do the job for you. You won't miss anything, but the interface won't be clogged with unused/inactive talkgroups.

**Reply by tadscottsmith** [reply]
> Again, you could let the autopopulate do the job for you. You won't miss anything, but the interface won't be clogged with unused/inactive talkgroups.

Autopopulate doesn't bring in any information? Autopopulated groups just show '-' for the name and label.

**Reply by chuot** [reply]
Does the json generated by trunk-recorder has a talkgroup_tag property?
